docs/release_notes/ReleaseNotes-2.8.0.md

```diff
@@ -35,8 +35,8 @@ See the wforce_webhook man page for more details.
 ## Enable IP and Login Substitution in blocklist return messages
 
 For example:
-setBlackistIPRetMsg("Go away your IP {ip} is blacklisted")
-setBlackistLoginRetMsg("Go away your login {login} is blacklisted")
+setBlacklistIPRetMsg("Go away your IP {ip} is blacklisted")
+setBlacklistLoginRetMsg("Go away your login {login} is blacklisted")
```

Comment:
running concern -- if the api was changed between releases then I'd expect there to be a note calling attention to this, so I'm hoping it's just been a typo in the docs...

Reply:
Just a typo